airbyte-integrations/connectors/source-mailchimp/acceptance-test-config.yml

```diff
@@ -33,24 +31,15 @@ acceptance_tests:
         empty_streams:
           - name: "automations"
             bypass_reason: "Cannot seed in free sandbox account, need to upgrade to paid account."
-      - config_path: "secrets/config_oauth.json"
```

Comment:
Why remove this?

Reply:
both configs (oauth and apikey) works with the same recordset. We already have a test_connection to ensure them.
In my opinion second test_read is excessive.